[pytorch] update llvmlite to 0.30.0

ljk53 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#31858 [pytorch] update llvmlite to 0.30.0**

Summary:
Trying to upgrade docker image but ran into the following error:

```
Running test_nn ... [2020-01-04 18:05:12.537860]
Traceback (most recent call last):
  File "test_nn.py", line 45, in <module>
    from common_cuda import TEST_CUDA, TEST_MULTIGPU, TEST_CUDNN, TEST_CUDNN_VERSION
  File "/var/lib/jenkins/workspace/test/common_cuda.py", line 16, in <module>
    import numba.cuda
  File "/opt/conda/lib/python3.6/site-packages/numba/__init__.py", line 178, in <module>
    _ensure_llvm()
  File "/opt/conda/lib/python3.6/site-packages/numba/__init__.py", line 100, in _ensure_llvm
    raise ImportError(msg)
ImportError: Numba requires at least version 0.30.0 of llvmlite.
Installed version is 0.28.0.
```

Differential Revision: [D19282923](https://our.internmc.facebook.com/intern/diff/D19282923)

ljk53 committed:
[pytorch] update llvmlite to 0.30.0

Summary:
Trying to upgrade docker image but ran into the following error:

```
Running test_nn ... [2020-01-04 18:05:12.537860]
Traceback (most recent call last):
  File "test_nn.py", line 45, in <module>
    from common_cuda import TEST_CUDA, TEST_MULTIGPU, TEST_CUDNN, TEST_CUDNN_VERSION
  File "/var/lib/jenkins/workspace/test/common_cuda.py", line 16, in <module>
    import numba.cuda
  File "/opt/conda/lib/python3.6/site-packages/numba/__init__.py", line 178, in <module>
    _ensure_llvm()
  File "/opt/conda/lib/python3.6/site-packages/numba/__init__.py", line 100, in _ensure_llvm
    raise ImportError(msg)
ImportError: Numba requires at least version 0.30.0 of llvmlite.
Installed version is 0.28.0.
```

[ghstack-poisoned]

ljk53 commented:
PR #31848 will make use of the docker image created by this PR.

ljk53 committed:
Update on "[pytorch] update llvmlite to 0.30.0"


Summary:
Trying to upgrade docker image but ran into the following error:

```
Running test_nn ... [2020-01-04 18:05:12.537860]
Traceback (most recent call last):
  File "test_nn.py", line 45, in <module>
    from common_cuda import TEST_CUDA, TEST_MULTIGPU, TEST_CUDNN, TEST_CUDNN_VERSION
  File "/var/lib/jenkins/workspace/test/common_cuda.py", line 16, in <module>
    import numba.cuda
  File "/opt/conda/lib/python3.6/site-packages/numba/__init__.py", line 178, in <module>
    _ensure_llvm()
  File "/opt/conda/lib/python3.6/site-packages/numba/__init__.py", line 100, in _ensure_llvm
    raise ImportError(msg)
ImportError: Numba requires at least version 0.30.0 of llvmlite.
Installed version is 0.28.0.
```

Differential Revision: [D19282923](https://our.internmc.facebook.com/intern/diff/D19282923)

[ghstack-poisoned]

ljk53 committed:
Update on "[pytorch] update llvmlite to 0.30.0"


Summary:
Trying to upgrade docker image but ran into the following error:

```
Running test_nn ... [2020-01-04 18:05:12.537860]
Traceback (most recent call last):
  File "test_nn.py", line 45, in <module>
    from common_cuda import TEST_CUDA, TEST_MULTIGPU, TEST_CUDNN, TEST_CUDNN_VERSION
  File "/var/lib/jenkins/workspace/test/common_cuda.py", line 16, in <module>
    import numba.cuda
  File "/opt/conda/lib/python3.6/site-packages/numba/__init__.py", line 178, in <module>
    _ensure_llvm()
  File "/opt/conda/lib/python3.6/site-packages/numba/__init__.py", line 100, in _ensure_llvm
    raise ImportError(msg)
ImportError: Numba requires at least version 0.30.0 of llvmlite.
Installed version is 0.28.0.
```

Differential Revision: [D19282923](https://our.internmc.facebook.com/intern/diff/D19282923)

[ghstack-poisoned]

kostmo commented:
## :pill: CircleCI build failures summary and remediations
As of commit fd33533c:


* **1/1** failures introduced in this PR


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=fd33533c8110e7fa34ca33121ee6b470ebd0a7e0).

### :detective: 1 new failure recognized by patterns
The following build failures do not appear to be due to upstream breakage:
#### [![See CircleCI build](https://avatars0.githubusercontent.com/ml/7?s=12)](https://circleci.com/gh/pytorch/pytorch/4139275) pytorch_linux_xenial_cuda9_cudnn7_py3_nogpu_test (1/1)
**Step:** "Test" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=80615817) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=80615817))
<details>
<summary>
<code>Jan 06 06:16:02 RuntimeError: test_dist_autograd_spawn failed!</code>
</summary>

```
Jan 06 06:16:01 Ran 49 tests in 165.526s 
Jan 06 06:16:01  
Jan 06 06:16:01 FAILED (errors=1, skipped=2) 
Jan 06 06:16:01  
Jan 06 06:16:01 Generating XML reports... 
Jan 06 06:16:02 Traceback (most recent call last): 
Jan 06 06:16:02   File "test/run_test.py", line 456, in <module> 
Jan 06 06:16:02     main() 
Jan 06 06:16:02   File "test/run_test.py", line 449, in main 
Jan 06 06:16:02     raise RuntimeError(message) 
Jan 06 06:16:02 RuntimeError: test_dist_autograd_spawn failed! 
Jan 06 06:16:02 + cleanup 
Jan 06 06:16:02 + retcode=1 
Jan 06 06:16:02 + set +x 
Jan 06 06:16:02 =================== sccache compilation log =================== 
Jan 06 06:16:02 ERROR:sccache::server: Compilation failed: Output { status: ExitStatus(ExitStatus(256)), stdout: "", stderr: "/tmp/torch_extensions/test_compilation_error_formatting/main.cpp: In function \'int main()\':\n/tmp/torch_extensions/test_compilation_error_formatting/main.cpp:2:23: error: expected \';\' before \'}\' token\n int main() { return 0 }\n                       ^\n" } 
Jan 06 06:16:02  
Jan 06 06:16:02 =========== If your build fails, please take a look at the log above for possible reasons =========== 
Jan 06 06:16:02 Compile requests                 61 
Jan 06 06:16:02 Compile requests executed        25 
Jan 06 06:16:02 Cache hits                       18 
```

</details>



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

ljk53 committed:
Update on "[pytorch] update llvmlite to 0.30.0"


Summary:
Trying to upgrade docker image but ran into the following error:

```
Running test_nn ... [2020-01-04 18:05:12.537860]
Traceback (most recent call last):
  File "test_nn.py", line 45, in <module>
    from common_cuda import TEST_CUDA, TEST_MULTIGPU, TEST_CUDNN, TEST_CUDNN_VERSION
  File "/var/lib/jenkins/workspace/test/common_cuda.py", line 16, in <module>
    import numba.cuda
  File "/opt/conda/lib/python3.6/site-packages/numba/__init__.py", line 178, in <module>
    _ensure_llvm()
  File "/opt/conda/lib/python3.6/site-packages/numba/__init__.py", line 100, in _ensure_llvm
    raise ImportError(msg)
ImportError: Numba requires at least version 0.30.0 of llvmlite.
Installed version is 0.28.0.
```

Differential Revision: [D19282923](https://our.internmc.facebook.com/intern/diff/D19282923)

[ghstack-poisoned]

ljk53 committed:
Update on "[pytorch] update llvmlite to 0.30.0"


Summary:
Trying to upgrade docker image but ran into the following error:

```
Running test_nn ... [2020-01-04 18:05:12.537860]
Traceback (most recent call last):
  File "test_nn.py", line 45, in <module>
    from common_cuda import TEST_CUDA, TEST_MULTIGPU, TEST_CUDNN, TEST_CUDNN_VERSION
  File "/var/lib/jenkins/workspace/test/common_cuda.py", line 16, in <module>
    import numba.cuda
  File "/opt/conda/lib/python3.6/site-packages/numba/__init__.py", line 178, in <module>
    _ensure_llvm()
  File "/opt/conda/lib/python3.6/site-packages/numba/__init__.py", line 100, in _ensure_llvm
    raise ImportError(msg)
ImportError: Numba requires at least version 0.30.0 of llvmlite.
Installed version is 0.28.0.
```

Differential Revision: [D19282923](https://our.internmc.facebook.com/intern/diff/D19282923)

[ghstack-poisoned]
